109 release notes (#217)
SolarArbiter/SolarArbiter.github.io: CHANGELOG.md

in the core documentation's [what's new](https://solarforecastarbiter-core.readthedocs.io/en/latest/whatsnew/index.html) series.

## [1.0.9] - 2021-12-03
{: .anchor}

### Added

- Outage tracking to the Solar Forecast Arbiter platform. Outages are tracked by 

>   a start and end time that can be used to exclude data from report analysis. Data

Reviewer: ```suggestion
  a start and end time that can be used to exclude data from report analyses. Data
```

  is excluded based on forecast issue times that fall within an outage. 

  Two varieties of outages exist:
  

>   -  System outages are defined by system administrators represent times when the

Reviewer: ```suggestion
  -  System outages are defined by system administrators and represent times when the
```

>      Solar Forecast Arbiter platform was unavailable. These can be utilized any

Reviewer: ```suggestion
     Solar Forecast Arbiter platform was unavailable. These can be used in any
```
